### ISSUE TITLE
[LS] Add type cast code action uses full module names for casting

### ISSUE THREAD

**Issue by IMS94** [ballerina-platform/ballerina-lang#28667]
**Description:**
Consider the following code snippet.
```
import ballerina/http;

 http:Listener server = new http:Listener(8080);
```
VSCode show a tool tip to cast the created listener to the expected type. Once its clicked, it changes that line as below:
```
 http:Listener server = <ballerina/http:1.1.0-alpha3:Listener> new http:Listener(8080);
```
where the full name of the module is used for casting.

https://user-images.githubusercontent.com/11285165/108057396-969a1980-7078-11eb-83a1-9d5840172039.mp4

**Steps to reproduce:**
See the attached screen capture.

**Affected Versions:**
SLA 3

**Comment by rasika** [comment]
Yes this is already fixed in latest master. This was due to FQPN pattern not being matching for `1.1.0-alpha3`.  https://github.com/ballerina-platform/ballerina-lang/pull/28651

**Comment by IMS94** [comment]
Great. No need of this issue (already addressed with #28651)
